lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1173,6 +1177,25 @@ def install_puppet_agent_dev_repo_on( hosts, opts )
             case variant
             when /^(fedora|el|centos|sles)$/
               on host, "rpm -ivh #{onhost_copied_file}"
+            when /^(aix)$/
+              # NOTE: AIX does not support repo management. This block assumes
+              # that the desired rpm has been mirrored to the 'repos' location.
+              #
+              # NOTE: tar is a dependency for puppet packages on AIX. So,
+              # we install it prior to the 'repo' file.
+              tar_pkg_path = "ftp://ftp.software.ibm.com/aix/freeSoftware/aixtoolbox/RPMS/ppc/tar"
```

Comment:
Installing tar should probably be something that we do during the configuration stage of beaker - but since we are under a time crunch I'll file a bug to move this around and we can go forward with the current patch.